providers/frame/utils/createCustomChainSpec.ts

```diff
@@ -0,0 +1,68 @@
+import { ss58, testUser } from "../../../crypto/mod.ts"
+
+const DEFAULT_TEST_USER_COUNT = 100_000
+const DEFAULT_TEST_USER_INITIAL_FUNDS = 1_000_000_000_000_000_000
+
+export interface CreateCustomChainSpecProps {
+  binary: string
+  chain: string
+  testUserAccountProps: {
```

Comment:
Same question about these props. Let's not preemptively parameterize this chain-spec writing util.

Reply:
ok